packages/backend/src/handlers/actions/guild/saveGuildHandler.ts

```diff
@@ -23,17 +31,37 @@ export const saveGuildHandler = async (
   const { input, session_variables: sessionVariables } = req.body;
   const playerId = sessionVariables['x-hasura-user-id'];
 
+  const { guildInformation } = input;
+
+  let updatedGuild: GuildFragment;
   try {
-    const { guildInformation } = input;
-    await saveGuild(playerId, guildInformation as GuildInfoInput);
-    res.json({ success: true });
+    updatedGuild = await saveGuild(
+      playerId,
+      guildInformation as GuildInfoInput,
+    );
```

Comment:
I'd move all the rest of this function into this `try` block. Since the error handler returns this is the same basic logic, but having `updatedGuild` outside the `try` makes it look as though it will be used afterwards which is only the case if there isn't an error.
```suggestion
    const updatedGuild = await saveGuild(
      playerId,
      guildInformation as GuildInfoInput,
    );
```

Reply:
Well, then we'd have nested try/catch blocks which I don't like. The catches need to be separate as one is fatal and the other can be ignored.